src/test/java/com/sigma/cinematicketpro/TestUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.sigma.cinematicketpro;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.sigma.cinematicketpro.dto.MovieDto;
+import com.sigma.cinematicketpro.entity.Movie;
+import com.sigma.cinematicketpro.mapper.MovieMapper;
+import com.sigma.cinematicketpro.mapper.impl.MovieMapperImpl;
+import org.modelmapper.ModelMapper;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.io.UnsupportedEncodingException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Optional;
+
+public final class TestUtils {
+    private final static ObjectMapper objectMapper = getObjectMapper();
+
+
+    private final static MovieMapper movieMapper = new MovieMapperImpl(new ModelMapper());
+
+    private TestUtils() {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+    }
+
+    public static ObjectMapper getObjectMapper() {
+        return Optional.ofNullable(objectMapper)
```

Comment:
why do you need all  this, just create new Objectmapper and register modules

Reply:
Made it static and left the getter